styles/styles.css

```diff
@@ -0,0 +1,462 @@
+    /* border: 1px solid rgba(0, 0, 0, 0.8); */
+    padding: 1rem;
+    font-size: 1rem;
+    text-align: center;
+  }
+
+  /* .flex-item{
+    width: 20%;
+    min-width: 20rem;
+    margin: 1rem;
+  } */
+
+  /* .flexbox-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  } */
+
+  /* article{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+  } */
+
+  article{
+    border-style: groove;
+  }
+
+  article img{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+    min-width: 100%;
+  }
+  article p{
+    text-align: center;
+  }
+  article h3{
+    text-align: center;
+  }
+  article h4{
+    text-align: center;
+  }
+
+  article button{
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 1rem 2rem;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 1rem;
+    margin: 4px 2px;
+    cursor: pointer;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  button{
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 1rem 2rem;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 1rem;
+    margin: 4px 2px;
+    cursor: pointer;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+
+.container {
+  width: 100%;
+  display: block;
+  margin: 0 auto;
+  /* background-color: peachpuff; */
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+@media screen and (min-width: 400px){
+  .box{
+    width: 100%;
+    border-style: groove;
+  }
+  main{
+    margin: 0 0 0 0;
+  }
+  /* img{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+    min-width: 100%;
+  }
+  p{
+    text-align: center;
+  }
+  h3{
+    text-align: center;
+  }
+  h4{
+    text-align: center;
+  }
+  button{
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 1rem 2rem;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 1rem;
+    margin: 4px 2px;
+    cursor: pointer;
+    margin-left: auto;
+    margin-right: auto;
+  } */
+}
+
+@media screen and (min-width: 768px) {
+  .box{
+    width: 45%;
+    border-style: groove;
+  }
+  article img{
+    min-width: 75%;
+  }
+  /* img{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+    min-width: 100%;
+  }
+  p{
+    text-align: center;
+  }
+  h3{
+    text-align: center;
+  }
+  h4{
+    text-align: center;
+  }
+  button{
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 1rem 2rem;
+    text-align: center;
+    text-decoration: none;
+    display: block;
+    font-size: 1rem;
+    margin: 4px 2px;
+    cursor: pointer;
+    margin-left: auto;
+    margin-right: auto;
+  } */
+}
+@media screen and (min-width: 1000px) {
+  .box{
```

Comment:
nice work implementing media queries, and just using min-width. It is not necessary to include both min and max widths. You broke up your screen sizes nicely!